Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volumegroup-snapshot.md

```diff
@@ -0,0 +1,100 @@
+---
+title: VolumeGroup Snapshots
+---
+
+## Prerequisites
+
+- Install the [snapshot controller and snapshot v1 CRD](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage).
```

Comment:
Pre req already mentioned in the kubernetes like you mentioned here https://github.com/kubernetes-csi/external-snapshotter/tree/master#volume-group-snapshot-support, just add a pointer to refer to the doc for all the pre-req